hazelcast/src/main/java/com/hazelcast/client/impl/ClientDelegatingFuture.java

```diff
@@ -382,6 +383,11 @@ public String toString() {
         return future.toString();
     }
 
+    // used for testing
+    V getDeserializedValue() {
```

Comment:
This method will be useful for testing of `DelegatingCompletableFuture` also, maybe it makes sense to pull it up?

Reply:
The problem is that then I cannot use a package-private access modifier. Is it OK to make it public?

Reply:
I don't think it's a problem, this class is internal API.